generate.sh

```diff
@@ -0,0 +1,29 @@
+mkdir temp
+
+#cp rep/doc temp/
+
+cd temp
+sphinx-quickstart -q -v $VERSION -a $AUTHOR -p $PROJECTNAME
+rm index.rst
+cd ..
+cp -a $DOCPATH. temp/
+cd temp
+make html
+git config --global user.email $EMAIL
```

Comment:
By generic, I mean not a specific CI name.
```
git config --global user.name "Bot"
git config --global user.email "noreply+bot@example.com"
```

It needs to be known it's an automated commit and it should not be linked to any person.